### PR TITLE
chapter_14

### PR DESCRIPTION
**Pull request by FirdausIsa**
ready

### CODE REVIEW

**Comment by FirdausIsa** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -54,10 +54,6 @@ def destroy
     redirect_to users_url
   end
 
-  def user_params
-    params.require(:user).permit(:name, :email, :password, :password_confirmation)
-  end
-
   def load_user
```
Comment (huongnt-2545) [inline app/controllers/users_controller.rb:57]: move method place to private scope

File: app/views/users/show_follow.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<% provide(:title, @title) %>
+
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <%= gravatar_for @user %>
+      <h1><%= @user.name %></h1>
+      <span><%= link_to "view my profile", @user %></span>
+      <span><b>Microposts:</b>
+        <%= @user.microposts.count %></span>
```
Comment (huongnt-2545) [inline app/views/users/show_follow.html.erb:10]: indent

**Comment by FirdausIsa** [comment]
ready

**Review by huongnt-2545 — COMMENTED** [review]
Still comment is not fix yet, why you resolve it??

**Comment by FirdausIsa** [comment]
I'm sorry I resolved the comment, I already unresolved it.  I already move the user params under private.

**Comment by FirdausIsa** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**